apps/judicial-system/web/src/routes/Court/Indictments/Overview/Overview.tsx

```diff
@@ -41,7 +38,7 @@ const IndictmentOverview = () => {
   const [modalVisible, setModalVisible] = useState<'RETURN_INDICTMENT'>()
 
   const latestDate = workingCase.courtDate ?? workingCase.arraignmentDate
-  const caseHasBeenReceivedByCourt = workingCase.state === CaseState.RECEIVED
+  // const caseHasBeenReceivedByCourt = workingCase.state === CaseState.RECEIVED
```

Comment:
This is intentionally commented out. This is used in the `resend indictment` functionality that has now been commented out.